### PR TITLE
Remove inline comments and variable substitution from .env_sample

### PR DESCRIPTION
**Pull request by rhymes**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

In the installation guide we tell people to copy `.env_sample` to `.env` but this file isn't actually valid, as numbers with inline comments will be interpreted as strings, see https://github.com/forem/forem/issues/13298#issuecomment-833590511 for example.

I decided to remove all inline comments from `.env_sample`

## Related Tickets & Documents

https://github.com/forem/forem/issues/13298#issuecomment-833590511

## Added tests?

- [ ] Yes
- [x] No, and this is why: not needed
- [ ] I need help with writing tests

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

- [ ] I've updated the [Developer Docs](https://docs.forem.com) and/or
      [Admin Guide](https://forem.gitbook.io/forem-admin-guide/), or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
- [ ] I've updated the README or added inline documentation
- [ ] I will share this change in a [Changelog](https://forem.dev/t/changelog)
      or in a [forem.dev](http://forem.dev) post
- [x] I will share this change internally with the appropriate teams
- [ ] I'm not sure how best to communicate this change and need help
- [ ] This change does not need to be communicated, and this is why not: _please
      replace this line with details on why this change doesn't need to be
      shared_

### CODE REVIEW

**Review by Zhao-Andy — APPROVED** [review]
Makes sense, nice catch!

File: .env_sample [hunk 1]
```diff
@@ -40,19 +41,27 @@ OPENRESTY_URL=""
...
 # Postgres DB
-DATABASE_NAME="Forem_development" # Used when creating a db instance `rails db:create`
-DATABASE_URL="postgresql://localhost:5432/$DATABASE_NAME" # This takes precedence over `DATABASE_NAME` if they don't match
-RAILS_MAX_THREADS=5 # Specifies the number of Puma threads
-WEB_CONCURRENCY=2 # Specifies the number of `workers` to boot in clustered mode.
-DATABASE_POOL_SIZE=5 # The number of concurrent connections to Postgres. If unset the value of RAILS_MAX_THREADS will be used in it's place.
+# Used when creating a db instance `rails db:create`
+DATABASE_NAME="Forem_development"
+# This takes precedence over `DATABASE_NAME` if they don't match
+DATABASE_URL="postgresql://localhost:5432/$DATABASE_NAME"
```
Comment (oyeyipowale) [inline .env_sample:47]: The variable $DATABASE_NAME should be substituted with the real value -> `Forem_development`. Same for the test db URL. Got an error too.

Comment (rhymes) [reply .env_sample:47]: Mmm that's weird, I just tried with `foreman` and the variable substitution works

Comment (Link2Twenty) [reply .env_sample:47]: Variable didn't work when building in wsl

Comment (rhymes) [reply .env_sample:47]: Ah! you're right! It works in the console, but it doesn't with Foreman

**Review by atsmith813 — APPROVED** [review]

**Review by Link2Twenty — APPROVED** [review]
LGTM

**Review by djuber — APPROVED** [review]
should we just remove the database name if the database url is used if present? 

It seemed like it _only_ existed to support the interpolation? Unless there's something happening in config/database.yml that contradicts what the comments in the sample env suggested.

**Comment by rhymes** [comment]
@djuber 

TLDR; I wouldn't touch them "in passing" in this PR

> It seemed like it only existed to support the interpolation? Unless there's something happening in config/database.yml that contradicts what the comments in the sample env suggested.

I'm not 100% sure but I think you're right, but there's a delicate balance I never remember about configuring `database.yml` (I alwayse use the URL for example) and the current settings were the product of a long lived PR I honestly forgot the context about: https://github.com/forem/forem/pull/11308
